charts/mattermost-enterprise-edition/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Mattermost Enterprise server with high availitibity.
 name: mattermost-enterprise-edition
-version: 0.7.1
+version: 0.7.2
```

it is a relative medium change would bump the minor 
```suggestion
version: 0.8.0
```